opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -36,10 +36,14 @@ def test_start_span_implicit(self):
 
             self.assertIsNotNone(root.start_time)
             self.assertIsNone(root.end_time)
+            self.assertEqual(root.kind, trace_api.SpanKind.INTERNAL)
 
-            with tracer.start_span("child") as child:
+            with tracer.start_span(
+                "child", kind=trace_api.SpanKind.CLIENT
+            ) as child:
                 self.assertIs(tracer.get_current_span(), child)
                 self.assertIs(child.parent, root)
+                self.assertEqual(root.kind, trace_api.SpanKind.INTERNAL)
```

Comment:
Shouldn't also be an assert on child.kind?

Reply:
Thx, fixed in 55f50fb75b25365f9ced5e67fd0f94291fd51afe (the assertion on root is above already, so this was a copy & paste error).